tests/Integration/Behaviour/Features/Context/Domain/Product/UpdateCategoriesFeatureContext.php

```diff
@@ -132,8 +136,8 @@ public function assertProductAssignedToDefaultCategory(string $productReference)
         $productCategoriesInfo = $productForEditing->getCategoriesInformation();
 
         $belongsToDefaultCategory = false;
-        foreach ($productCategoriesInfo->getCategoryIds() as $categoryId) {
-            if ($categoryId === $defaultCategoryId) {
+        foreach ($productCategoriesInfo->getCategoriesInformation() as $categoryInformation) {
```

Comment:
Since now `CategoryInformation` also returns the names of the categories it would nice to update the behat tests accordingly And check not only the id references but also the returned names

Reply:
Not sure how :thinking: 
The case we have is that we only assign the categories to product, so we only need id (or the id reference in behat)
```
    When I assign product product1 to following categories:
      | categories       | [home, men, clothes] |
      | default category | clothes              |
```
And what we want to assert is only the result object CategoryInformation inside ProductForEditing. Only solution I see for now is if we compare Category object model names with the CategoryInformation names. This could be done without needing to change behat feature definitions

Reply:
implemented assertion with localized names in latest commits :ok_hand: